php/countries.php

```diff
@@ -5,12 +5,8 @@
 
 // List of all countries
 $sql = "SELECT iso_code AS code, name FROM countries ORDER BY name";
-$first = true;
+$rows = [];
 foreach ($dbh->query($sql) as $row) {
-    if ($first) {
-        $first = false;
-    } else {
-        printf("\n");
-    }
-    printf("%s;%s", $row["code"], $row["name"]);
+    $rows[] = sprintf("%s;%s", $row["code"], $row["name"]);
 }
+echo implode(sprintf("\m"), $rows);
```

Comment:
That's clearly wrong. Needs to be \n and the sprintf can go away too like the other calls

Reply:
This endpoint isn't used by the website at all and since we don't have a public API, I think we can just remove it.

Reply:
It has tests!!!!!